soroban-env-common/env.json

```diff
@@ -1382,7 +1382,7 @@
                         }
                     ],
                     "return": "Void",
-                    "docs": "If the entry's TTL is below `threshold` ledgers, extend `live_until_ledger_seq` such that TTL == `extend_to`, where TTL is defined as live_until_ledger_seq - current ledger"
+                    "docs": "If the entry's TTL is below `threshold` ledgers, extend `live_until_ledger_seq` such that TTL == `extend_to`, where TTL is defined as live_until_ledger_seq - current ledger. If attempting to extend the entry past the maximum allowed value (defined as the current ledger + `max_entry_ttl` - 1), and the entry is `Persistent`, its new `live_until_ledger_seq` will be clamped to the max; if the entry is `Temporary`, an error is returned."
```

Comment:
Env doesn't return an error, it traps. Please update the doc.

Reply:
Fixed.